packages/hint-compat-api/tests/html-next.ts

```diff
@@ -0,0 +1,166 @@
+import { getHintPath } from 'hint/dist/src/lib/utils/hint-helpers';
+import generateHTMLPage from 'hint/dist/src/lib/utils/misc/generate-html-page';
+import readFile from 'hint/dist/src/lib/utils/fs/read-file';
+import { HintTest } from '@hint/utils-tests-helpers/dist/src/hint-test-type';
+import * as hintRunner from '@hint/utils-tests-helpers/dist/src/hint-runner';
+
+const hintPath = getHintPath(__filename, true);
+
+const generateHTMLConfig = (fileName: string) => {
+    const path = 'fixtures/html';
+    const htmlFile = readFile(`${__dirname}/${path}/${fileName}.html`);
+
+    return { '/': generateHTMLPage(htmlFile) };
+};
+
+/*
+ * Tests for html features that were removed / deprecated.
+ * More information about how `hintRunner` can be configured is
+ * available in:
+ * https://webhint.io/docs/contributor-guide/how-to/test-hints/
+ */
+
+const elementAddedAlwaysTrue: HintTest[] = [
+    {
+        name: 'Elements that have added as true should pass.',
+        serverConfig: generateHTMLConfig('div')
+    }
+];
+
+hintRunner.testHint(hintPath, elementAddedAlwaysTrue, { browserslist: ['last 2 Edge versions'], parsers: ['html']});
```

Comment:
Remove the explicit HTML parser from this and all other tests and the double-events issue (one from `parser-html` and one from the connector) should go away. We don't need the HTML parser here because both connectors we use in the test suite (`jsdom` and `chrome`) will generate `element::*` events on their own.

```suggestion
hintRunner.testHint(hintPath, elementAddedAlwaysTrue, { browserslist: ['last 2 Edge versions']});
```